### PR TITLE
Upgrade to TensorFlow 2.0.0

### PR DESCRIPTION
**Pull request by shendiaomo**
This is identical to #46 but pulls from a branch to pass CI

### CODE REVIEW

File: sqlflow_models/dnnclassifier.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import tensorflow as tf
 
 class DNNClassifier(tf.keras.Model):
-    def __init__(self, feature_columns=None, hidden_units=[10,10], n_classes=3):
+    def __init__(self, feature_columns=None, hidden_units=[100,100], n_classes=3):
```
Comment (tonyyang-svail) [inline sqlflow_models/dnnclassifier.py:4]: Why updating default values?

File: .travis.yml [hunk 2]
```diff
@@ -12,6 +12,9 @@ python:
 service:
 - docker
 install:
+- python -m pip install --upgrade pip
+- python -m pip install --upgrade setuptools>=41.0.0
+- python -m pip install --upgrade tensorflow==2.0.0
```
Comment (tonyyang-svail) [inline .travis.yml]: Following up the discussion at https://github.com/sql-machine-learning/models/pull/46#discussion_r364054604.

Hi @shendiaomo, Please allow me to explain.

I understand that line 15 and line 16 might be necessary due to pip packages in Travis CI being outdated. However, I am questioning the nececity of line 17.

File: sqlflow_models/dnnclassifier.py (hunk 1 above)
Comment (shendiaomo) [reply sqlflow_models/dnnclassifier.py:4]: 1. To make the unit test stably meet the loss criterion.
1. A network of `[10, 10]` is less than useless for almost all real-world datasets.

Comment (tonyyang-svail) [reply sqlflow_models/dnnclassifier.py:4]: I see. Thanks.

**Review by tonyyang-svail — APPROVED** [review]

**Comment by typhoonzero** [comment]
Thanks for this work
